portfolio/src/main/webapp/script.js

```diff
@@ -48,3 +48,12 @@ function createListElement(text) {
   liElement.innerText = text;
   return liElement;
 }
+
+
+function createMap() {
+  const map = new google.maps.Map(document.getElementById("map"), {
+    center: { lat: 29.646445, lng: -82.347676 },
+    zoom: 16
```

Comment:
[Style](https://google.github.io/styleguide/jsguide.html#features-objects-use-trailing-comma): Include a trailing comma whenever there is a line break between the final property and the closing brace.

```suggestion
    zoom: 16,
```